controllers/albums.js

```diff
@@ -95,17 +96,34 @@ const get_album_by_id = asyncHandler(async (req, res, next) => {
     albumId: req.params.albumId,
   };
 
-  const album = await prisma.album.findFirstOrThrow({
-    where: { id: request.albumId },
-    // include artist.userId at the top level of the album
-    include: {
-      artist: {
-        select: {
-          userId: true,
+  if (!validate(request.albumId)) {
+    res.status(400).json({
+      success: false,
+      error: "Invalid albumId",
+    });
+    return;
+  }
+
+  const album = await prisma.album
+    .findFirstOrThrow({
+      where: { id: request.albumId },
+      // include artist.userId at the top level of the album
+      include: {
+        artist: {
+          select: {
+            userId: true,
+          },
         },
       },
-    },
-  });
+    })
+    .catch((err) => {
+      // Prisma will throw an error if the uuid is not found or not a valid uuid
+      res.status(400).json({
+        success: false,
+        error: "No album found with that id",
+      });
+      return;
+    });
```

Comment:
Just realized this will return out of the `.catch` and continue executing the code below it. So we need to check if album is defined, and return if it isn't. 
```suggestion
      return;
    });

  if (!album) {
    return;
  }
```